plugins/catalog-backend-module-aws/src/processors/AwsEKSClusterProcessor.ts

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright 2022 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import {
+  CatalogProcessor,
+  CatalogProcessorEmit,
+  LocationSpec,
+} from '@backstage/plugin-catalog-backend';
+import { Credentials, EKS } from 'aws-sdk';
+import { AWSCredentialFactory } from '../types';
+
+const ACCOUNTID_ANNOTATION: string = 'amazonaws.com/account-id';
+const ARN_ANNOTATION: string = 'amazonaws.com/arn';
+const KUBERNETES_API_SERVER_ANNOTATION = 'kubernetes.io/api-server';
+const KUBERNETES_API_SERVER_CA_ANNOTATION =
+  'kubernetes.io/api-server-certificate-authority';
+const KUBERNETES_AUTH_METHOD_ANNOTATION = 'kubernetes.io/auth-provider';
+
+export class AwsEKSClusterProcessor implements CatalogProcessor {
```

Comment:
I don't see any api-report change for this package - did you forget to export it all the way?